docker/test-server/etebase-server.ini

```diff
@@ -3,10 +3,19 @@ secret_file = secret.txt
 debug = true
 static_root = /data/static
 media_root = /data/media
+AWS_ACCESS_KEY_ID = AKIARXDELOGPTJKCHC5B
+AWS_SECRET_ACCESS_KEY = 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
+AWS_STORAGE_BUCKET_NAME = find-etebase-store-dev
 
 [allowed_hosts]
 allowed_host1 = *
 
 [database]
-engine = django.db.backends.sqlite3
-name = /data/db.sqlite3
+ENGINE = django.db.backends.postgresql_psycopg2
+NAME = eteserverdb
+USER = root
+PASSWORD = mysecretpassword
```

Comment:
Things like db user and password should be env variables

Reply:
Again, this file is used to store env variables in this repo, if you want to use .env file, then I can add/replace it with that as well. 
What you say?

Reply:
@vitaly512 yes we should update to use env vars for all these secrets.

Reply:
You can create a .env.example file to show what envs are needed. You can also create your own README file.

Reply:
Okay, I will update that.